src/makers/peoples/antennas.ts

```diff
@@ -0,0 +1,8 @@
+import { MakerAntenna } from '../antennaMakerTypes'
+
+const peoples = {
+  name:'"People\'s Antenna" - PPL',
```

Comment:
There is a lint error here. It should be `  name: '"People\'s Antenna" - PPL',`

Reply:
You may also want to remove the `"` so that this will be in the proper alphabetical order.

Reply:
Fixed, thanks!